db/write_controller.cc

```diff
@@ -47,16 +47,16 @@ uint64_t WriteController::GetDelay(Env* env, uint64_t num_bytes) {
     return 0;
   }
 
-  const uint64_t kMicrosPerSecond = 1000000;
-  const uint64_t kRefillInterval = 1024U;
+  const uint64_t kNanosPerSecond = 1000000000U;
+  const uint64_t kRefillInterval = 1000000U;
```

Comment:
Yes we can. I chose to use 1000000 because I think it won't be any performance degradation in practical cases. And it makes the calculation for creating the tests much easier.